group_vars/all.yml.sample

```diff
@@ -273,6 +273,16 @@ dummy:
 
 #cephx: true
 
+# Cluster configuration
+#ceph_cluster_conf:
+#  global:
+#    public_network: "{{ public_network | default(omit) }}"
+#    cluster_network: "{{ cluster_network | default(omit) }}"
+#    osd_pool_default_crush_rule: "{{ osd_pool_default_crush_rule }}"
+#    ms_bind_ipv6: "{{ (ip_version == 'ipv6') | string }}"
+#    ms_bind_ipv4: "{{ (ip_version == 'ipv4') | string }}"
+#    osd_crush_chooseleaf_type: "{{ '0' if common_single_host_mode | default(false) else omit }}"
```

Comment:
```suggestion
# ceph_cluster_conf:
#   global:
#     public_network: "{{ public_network | default(omit) }}"
#     cluster_network: "{{ cluster_network | default(omit) }}"
#     osd_pool_default_crush_rule: "{{ osd_pool_default_crush_rule }}"
#     ms_bind_ipv6: "{{ (ip_version == 'ipv6') | string }}"
#     ms_bind_ipv4: "{{ (ip_version == 'ipv4') | string }}"
#     osd_crush_chooseleaf_type: "{{ '0' if common_single_host_mode | default(false) else omit }}"
```

Reply:
This is generated by the script - would it work by changing it? I guess CI would complain?

Reply:
We have to modify the script then 🙂

Reply:
I guess it was good to have comments for vars with no space so the real comments for vars could be distinguished from those. What do you think?